META-INF/MANIFEST.MF

```diff
@@ -1,4 +1,4 @@
 Manifest-Version: 1.0
 Ant-Version: Apache Ant 1.9.6
-Created-By: 1.8.0_191-b12 (Oracle Corporation)
+Created-By: 1.8.0_91-b14 (Oracle Corporation)
```

Comment:
hm, i would keep the original file or .gitignore it if it is generated. but downgrading to such an old version looks strange.

Reply:
OK